src/rsz/src/Resizer.cc

```diff
@@ -497,18 +498,31 @@ void Resizer::balanceRowUsage()
 
 ////////////////////////////////////////////////////////////////
 
-void Resizer::findBuffers()
+void Resizer::findDataBuffers()
 {
   if (buffer_cells_.empty()) {
     LibertyLibraryIterator* lib_iter = network_->libertyLibraryIterator();
+    sta::PatternMatch clkbuf_pattern(".*CLKBUF.*",
+                                     /* is_regexp */ true,
+                                     /* nocase */ true,
+                                     /* Tcl_interp* */ nullptr);
```

Comment:
It feels like we should factor this out into a separate function. There's lots and lots of different names for these clock buffers.

In general though it feels like we should be able to identify them based on the difference between their rise and fall times. We should have one class of buffers where the delta is much bigger and one class where it's nearly zero. If we run a k-means of 2 I suspect we'll get what we want.

Reply:
I used as reference the strategy used in TritonCTS when we select the buffers that will be used for the clock tree.
See `TritonCTS::inferBufferList`. I opened this as a draft to propose a discussion as I don't feel that this change is close to an actual solid solution. @maliberty @precisionmoon Do you have any input?

Reply:
@QuantamHD, @AcKoucher, Yes it makes sense to look for buffers with similar rise/fall delays in the absence of special buffer names or attributes.   We want to do this for both CTS and RSZ engines.  For now, I suggest that we proceed with cell names and attributes and enhance it for rise/fall delays in the future.

Reply:
I looked and all the public PDKs would work with this (not all have clock buffers).  There are private PDKs that don't follow this pattern.  I agree it is ok for a first step.

Reply:
We could centralize the decision about what is a clock buffer in dbSta so it can be shared with CTS.

Reply:
It seems like proceeding with a half solution like the clockbuf pattern thing would be a poor idea in my mind. If the better solution is the look at rise and fall times, etc then this should wait until a centralize solution is available